docs/app/client/carousel/examples/SelectIndex.tsx

```diff
@@ -22,8 +22,14 @@ const SelectIndex = () => {
           </Chip>
         ))}
       </div>
-      <Carousel step={1} selectedIndex={selected} className="w-[320px]">
-        <Carousel.Reel className="overflow-x-hidden">
+      <Carousel
+        step={1}
+        selectedIndex={selected}
+        className="w-[320px]"
```

Comment:
Please, don't use arbitrary values in classes. w-80 is more native TW class for that. And what about iPhone SE. Will Carousel fit into the view?

Reply:
Updated to use tailwind utility class and to fit on iPhone SE. Same on other occurrences.